### ISSUE TITLE
package.json : Shouldn't main point to app/main.js ?

### ISSUE THREAD

**Issue by hepiyellow** [chentsulin/electron-react-boilerplate#924]
Hi,

Shouldn't the `main` property in `package.json` point to `app/main.js` instead of `main.js`?

**Comment by othke** [comment]
Hi @hepiyellow ,

I think the app/ redirection is made by the `start` script

> "start": "cross-env NODE_ENV=production electron ./app/",

**Comment by hepiyellow** [comment]
Well, sorry for the un-informative question.

Here is my real question:
I added this VS Code launch configuration:
```
{
            "name": "Launch",
            "type": "node",
            "request": "launch",
            "program": "${workspaceRoot}/app/main.js",
            "stopOnEntry": false,
            "args": [],
            "cwd": "${workspaceRoot}",
            "preLaunchTask": null,
            "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron",
            "runtimeArgs": [
                ".",
                "--enable-logging"
            ],
            "env": {},
            "console": "internalConsole",
            "sourceMaps": false,
            "outDir": null
        }
```

And it works only if I change `package.json`'s `main` to be `app/main.js` instead of `main.js`.
When it is only `main.js` then I get pop-up window with an error saying:
"unable to find Electron app at [path to my electron-react-boilerplate]"

Is there something wrong in my launch config?
 
Using VS Code 1.11.2

**Comment by amilajack** [comment]
I closed this because there wasn't a response from you so I assumed you resolved this. My bad. Try setting it to `app/main.development.js`. I'm just guessing. I haven't actually configured vscode with electron before

**Comment by hepiyellow** [comment]
VS code looks for the `main` property in `package.json`.

I found this: 
http://electron.rocks/debugging-electron-in-vs-code/

And it says:
"program - Your app entry point - same as main in package.json (needs to be explicit, setting "." won't do)"

Solution:
It also says for the launch config:
"cwd - Path to the program being debugged (in our case, that's root where program file is)"

So I set:  
`"cwd": "${workspaceRoot}/app",`
instead of : 
"cwd": "${workspaceRoot}",

and it works now.

I don't know if there are other implications of having the working directory at `app/`,
but for now it works.
